webpack/assets/javascripts/react_app/components/SettingsTable/components/SettingCellInner.js

```diff
@@ -26,30 +19,21 @@ const SettingCellInner = props => {
     setModalOpen();
   };
 
-  const field = (
-    <span
-      onClick={editable ? openModal : undefined}
-      {...rest}
-      className={cssClasses}
-    >
+  let field = (
+    <div onClick={editable ? openModal : undefined} {...rest}>
       {valueToString(setting)}
-    </span>
+    </div>
   );
 
-  const value =
-    setting.value !== setting.default && hasDefault(setting)
-      ? inStrong(field)
-      : field;
-  return <EllipsisWithTooltip>{value}</EllipsisWithTooltip>;
+  if (setting.value !== setting.default && hasDefault(setting))
+    field = <strong>{field}</strong>;
+  return field;
 };
 
 SettingCellInner.propTypes = {
   setting: PropTypes.object.isRequired,
-  className: PropTypes.string,
 };
 
-SettingCellInner.defaultProps = {
-  className: '',
-};
+SettingCellInner.defaultProps = {};
```

Comment:
empty default props aren't needed

Reply:
oops, must have missed this in the rebase, fixed now. Is there any eslint rule we can enable to prevent this?

Reply:
I haven't found any yet